### PR TITLE
Swim trap

### PR DESCRIPTION
**Pull request by jlyden**
Please review this, @casey-collab

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2669)

### CODE REVIEW

File: english/dream/swim/swim.md [hunk 1]
```diff
@@ -11,3 +11,15 @@ A pain shoots up your legs. You glance down to where your feet should be only to
...
+
+... Ursula the sea witch!
+
+She is enormous, with a mouth large enough to eat you in three bites! Her licorice-whip hair is 
+wrapped around your flipper, and she begins to pull you towards her, reeling in the licorice like 
+fishing line.
+
+What to do!?
+
+[Attempt to chew through the licorice lines ensnaring you.](/english/dream/swim/swim-trap/chew.md)
```
Comment (casey-collab) [inline english/dream/swim/swim.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/dream/swim/swim.md [hunk 2]
```diff
@@ -11,3 +11,15 @@ A pain shoots up your legs. You glance down to where your feet should be only to
...
+[Pinch yourself to see if this is a dream.](./is-this-a-dream/is-this-a-dream.md)
```
Comment (casey-collab) [inline english/dream/swim/swim.md]: This link is also invalid.

File: english/dream/swim/swim.md [hunk 3]
```diff
@@ -11,3 +11,15 @@ A pain shoots up your legs. You glance down to where your feet should be only to
...
+[Remember that you are superman.](./remember-you-are-superman/remember-you-are-superman.md)
```
Comment (casey-collab) [inline english/dream/swim/swim.md]: This link is also invalid.

File: english/dream/swim/swim.md (hunk 2 above)
Comment (casey-collab) [inline english/dream/swim/swim.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

File: english/dream/swim/swim.md (hunk 3 above)
Comment (casey-collab) [inline english/dream/swim/swim.md]: This link is also on the line directly after another link.

File: english/dream/swim/swim-trap/chew.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+As Ursula the Sea Witch pulls you closer, her licorice-whip hair wrapped around your gummy fin,
```
Comment (casey-collab) [inline english/dream/swim/swim-trap/chew.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**